components/search/common.tsx

```diff
@@ -27,3 +28,31 @@ export function getServerConfig(): TypesenseInstantsearchAdapterOptions["server"
     ]
   }
 }
+
+function RefinementList({ attribute }: { attribute: string }) {
+  useRefinementList({ attribute })
+  return null
+}
+
+export function VirtualFilters({ type }: { type: "bill" | "testimony" }) {
+  const refinementAttributes =
+    type === "testimony"
+      ? ["authorDisplayName", "court", "position", "billId", "authorRole"]
+      : [
+          "court",
+          "currentCommittee",
+          "city",
+          "primarySponsor",
+          "cosponsors",
+          "topics.lvl0",
+          "topics.lvl1"
+        ]
+
+  return (
+    <>
+      {refinementAttributes.map(attribute => (
+        <RefinementList key={attribute} attribute={attribute} />
```

Comment:
Not something that necessarily needs to be fixed in this PR - but using a `RefinementList` for the `HierarchicalMenu` attributes `topics.lvl0`/`topics.lvl1` may complicate the logic around those filters down the road.

This is still a strict improvement over current behavior so I think we should merge before fixing this - just wanted to call this out.